ionoscloud/data_source_nic_test.go

```diff
@@ -72,6 +77,12 @@ func Test_dataSourceNicRead(t *testing.T) {
 	if *nic.Properties.Dhcp != data.Get("dhcp").(bool) {
 		t.Fatalf("expected '%t', got '%s'", *nic.Properties.Dhcp, data.Get("dhcp"))
 	}
+	if *nic.Properties.Dhcpv6 != data.Get("dhcpv6").(bool) {
+		t.Fatalf("expected '%t', got '%s'", *nic.Properties.Dhcp, data.Get("dhcp"))
+	}
+	if *nic.Properties.Ipv6CidrBlock != data.Get("ipv6_cidr_block").(string) {
+		t.Fatalf("expected '%t', got '%s'", *nic.Properties.Dhcp, data.Get("dhcp"))
```

Comment:
wrong err message, it's not dhcp that's being checked

Reply:
corrected typo